ThunderstoreCLI/Config/Config.cs

```diff
@@ -140,19 +137,23 @@ public static Config Parse(params IConfigProvider[] configProviders)
         public static void Merge<T>(T target, T source, bool overwrite)
         {
             if (source == null)
+            {
                 return;
+            }
 
-            var t = typeof(T);
-            var properties = t.GetProperties();
+            Type? t = typeof(T);
```

Comment:
Did everything default to being Nullable after being converted from var? This expression cannot evaluate as null and yet the type is still nullable, with several more expressions like this existing throughout the PR

Reply:
> Did everything default to being Nullable after being converted from var?

Yes, the IDE did this automatically. I thought it was odd as well hmm

Reply:
Just to clarify, it is evaluated as Type? even when using var

![image](https://user-images.githubusercontent.com/844685/141615008-f6f09a1b-993d-4069-ada3-ff9e6d789a3f.png)

I guess this is because of the `<Nullable>enable</Nullable>` property in the .csproj but I'm not positive